### PR TITLE
Csv generator and sftp

### PR DESCRIPTION
**Pull request by saneshark**
## Description of change
Updates the CSV Generator and SFTP

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        end
+      end
+
+      private
+
+      def remote_file_path(name)
+        "/#{name}"
+      end
+
+      def generated_file_name
```
Comment (patrickvinograd) [inline modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb:30]: Can you make it so DHS_load is a default param and if pass in something else like TEST_ it doesn't even include 'DHS_load', that way if their pattern matching is not great, a TEST file won't have that string anywhere in it.

Comment (patrickvinograd) [reply modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb:30]: Maybe that's over-thinking it

**Review by patrickvinograd — APPROVED** [review]
LGTM

File: modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb (hunk 1 above)
Comment (saneshark) [reply modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb:30]: I think he said if it's prefixed with anything other than DHS_load it wont pick up automatically

Comment (patrickvinograd) [reply modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb:30]: nevermind, but we can make sure whatever rake tasks we put in place are safe w/r/t filenames

**Pull request merged**